core/themes/seven/css/style.css

```
@@ -652,85 +652,89 @@ tr.selected td {
  *   to apply all padding to the inner .fieldset-wrapper instead.
  */
 fieldset {
+  position: relative;
   max-width: 100%;
   margin: 1em 0;
   padding: 0;
+  border-radius: 4px;
   background-color: #fff;
-  border: none;
+  border: 2px solid #EAEAEA;
   min-width: 0;
 }
-
 fieldset .fieldset-legend {
+  position: relative;
+  left: 0; /* LTR */
+  top: 0;
   display: block;
+  width: 100%;
+  padding-left: 15px; /* LTR */
   box-sizing: border-box;
-  border: 2px solid #EAEAEA;
-  border-radius: 4px;
-  background-color: #fff;
-}
-fieldset:not(.collapsible) .fieldset-legend {
-  padding: .3em .3em .3em 1em; /* LTR */
 }
-[dir="rtl"] fieldset:not(.collapsible) .fieldset-legend {
-  padding: .3em 1em .3em .3em;
+fieldset:not(.collapsible) > legend > .fieldset-legend {
+  top: .35em;
 }
-fieldset:not(.collapsible) .fieldset-legend,
-fieldset.collapsible:not(.collapsed) .fieldset-legend {
-  border-bottom: none;
+[dir="rtl"] fieldset .fieldset-legend {
+  right: 0;
+  padding-right: 15px;
+  padding-left: 0;
 }
 legend {
   font-size: 120%;
-  font-weight: normal;
+  float: left;
   width: 100%;
 }
-
-fieldset .fieldset-wrapper {
-  padding: 15px 13px 13px 15px; /* LTR */
-  border-radius: 0 0 4px 4px;
-  border: 2px solid #EAEAEA;
-  border-top: none;
-  margin-top: -2px;
-}
-[dir="rtl"] fieldset .fieldset-wrapper {
-  padding: 15px 15px 13px 13px;
-}
-.vertical-tabs fieldset .fieldset-wrapper {
-  border: none;
+[dir="rtl"] legend {
+  float: right;
 }
-
 .fieldset-description {
   margin-bottom: 1em;
 }
-
 fieldset fieldset {
   background-color: #fff;
 }
 fieldset fieldset fieldset {
   background-color: #f8f8f8;
 }
-
 /**
  * Collapsible Fieldsets
  */
+.js fieldset.collapsible {
+  position: relative;
+}
+fieldset .fieldset-wrapper {
+  padding: 0 13px 13px 15px; /* LTR */
+  clear: both;
+}
+[dir="rtl"] fieldset .fieldset-wrapper {
+  padding: 0 15px 13px 13px;
```

I think it might be better to specify the left/right values separately here, to avoid duplicating the top/bottom values:

```suggestion
  padding-right: 15px;
  padding-left: 13px;
```

---

Are you sure? There are loads of such css code (see examples in same file). This doesn't do any harm, is not wrong and does not violate any standard (to my knowledge).

---

No, not so sure about this. Just something I noticed at first, but then noticed the rest of the code as you mentioned. So I'm not set against having all values specified here, it was just something I thought would be nice to avoid duplication. But I guess that's for another issue...